### PR TITLE
add feature flag to enable the relevant period feature

### PR DESCRIPTION
**Pull request by ConnorOS-ch**
### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 1]
```diff
@@ -32,3 +32,4 @@ feature_flag_disable_update_private_data_fetch_28062023 = "false"
 feature_flag_enable_update_manage_trusts_29082023 = "true"
 feature_flag_enable_redis_removal_27092023 = "true"
 feature_flag_enable_cease_trusts_19022024 = "false"
+feature_flag_enable_relevant_period_26042024 = "false"
```
Comment (mzacharias-ch) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:35]: should not this be true? as we want the changes see in development, staging env.

**Review by mzacharias-ch — CHANGES_REQUESTED** [review]

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 1 above)
Comment (ConnorOS-ch) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:35]: potentially, I'm not a Terraform expert, so at the moment I just went for turned off in all envs except our local. Probably we can ask dev ops to turn these on as and when we need to use each for testing.

Comment (markpit) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:35]: With the terraform approach on the web services, the pipeline will pick up the terraform from your PR and push it through and apply it to CI-Dev. When the next release goes into Staging and live, it takes the latest terraform with it. So if you leave it as false, but want it set to true for staging, you would need to create a new PR to set it to true when you are ready to go to staging.
For the services that haven't migrated to these new Amazon ECS pipelines (like overseas-entities-api) you would request the release team manually change the config in staging and live via release notes.
The Pr looks ok to me if you are happy with the config values.

**Review by mwestacott — APPROVED** [review]
PR looks fine to me. Approving but note that this code shouldn't be merged to `main` until the fees changes have gone live next Tuesday.

**Review by mzacharias-ch — APPROVED** [review]

**Pull request merged**